features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/db2/action-cleanup/db2-stale-action-cleanup.sql

```diff
@@ -0,0 +1,117 @@
+CREATE OR REPLACE PROCEDURE WSO2_IDN_ACTION_CLEANUP
+
+BEGIN
+
+-- ------------------------------------------
+-- DECLARE VARIABLES
+-- ------------------------------------------
+DECLARE batchSize INT;
+DECLARE chunkSize INT;
+DECLARE batchCount INT;
+DECLARE chunkCount INT;
+DECLARE rowCount INT;
+DECLARE v_rowCount INT;
+DECLARE enableLog SMALLINT;
+DECLARE backupTables SMALLINT;
+DECLARE SQLStatement VARCHAR(100);
+
+-- ------------------------------------------
+-- CONFIGURABLE VARIABLES
+-- ------------------------------------------
+SET batchSize    = 10000;
+SET chunkSize    = 500000;
+SET enableLog    = 0;
+SET backupTables = 0;
+
+SET rowCount = 0;
+SET batchCount = 1;
+SET chunkCount = 1;
+
+IF (enableLog = 1) THEN
+	CALL DBMS_OUTPUT.PUT_LINE('WSO2_ACTION_CLEANUP() STARTED...!');
```

Comment:
Should we say WSO2_ACTION I think this info should be more descriptive about what we are cleaning